numpy/distutils/mingw32ccompiler.py

```diff
@@ -141,17 +146,16 @@ def __init__ (self,
                     compiler='gcc -mno-cygwin -O2 -Wall',
                     compiler_so='gcc -mno-cygwin -O2 -Wall'
                                 ' -Wstrict-prototypes',
+                    compiler_cxx='g++',
                     linker_exe='g++ -mno-cygwin',
                     linker_so='g++ -mno-cygwin -shared')
             else:
                 # gcc-4 series releases do not support -mno-cygwin option
                 self.set_executables(compiler='gcc -O2 -Wall',
                                      compiler_so='gcc -O2 -Wall -Wstrict-prototypes',
+                                     compiler_cxx='g++',
                                      linker_exe='g++ ',
                                      linker_so='g++ -shared')
-        # added for python2.3 support
-        # we can't pass it through set_executables because pre 2.2 would fail
-        self.compiler_cxx = ['g++']
```

Comment:
It looks like this is maybe still not supported by MSVCCompiler

`AttributeError: 'MSVCCompiler' object has no attribute 'ldflags_shared'`